time timestamp
bool estopTriggered
bool dataOk
